userguide/analytics/userguide_analytics.adoc

```diff
@@ -1,63 +1,595 @@
-= Kill Bill analytics guide
+= Kill Bill Analytics Guide
 
-This is the reference guide to get started with https://github.com/killbill/killbill-analytics-plugin[Kill Bill analytics plugin].
+The https://github.com/killbill/killbill-analytics-plugin[Kill Bill analytics plugin] provides business analytics and reporting capabilities. You can use this plugin to create reports and view them via Kaui. Not only that, it allows creating reports against external databases as well. This document explains how to install and use this plugin. 
+
+== Prerequisites
+
+* Ensure that you have Kill Bill, Kaui, and the database set up as explained in the https://docs.killbill.io/latest/getting_started.html[Getting Started Guide].
+
+* Ensure that you have https://curl.haxx.se/[cURL] installed. If you are on Windows, we recommend that you use https://git-scm.com/download/win[Git Bash] to run the `cURL` commands.
 
 
 [[components-analytics-overview]]
-== Overview
+== Analytics Plugin Internals
+
+Kill Bill maintains a separate <<analytics_tables, reporting database>> for analytics purposes. The tables in this database are meant to be used as building blocks for reports. In contrary to other tables in the system, this set of tables is stable and potentially changes only between major revisions of Kill Bill. This database is kept current in realtime. The https://github.com/killbill/killbill-analytics-plugin[analytics plugin] is a https://docs.killbill.io/latest/notification_plugin.html[notification plugin]. It listens to https://docs.killbill.io/latest/kill_bill_events.html[events] generated by Kill Bill when any system change occurs and rebuilds the analytics tables upon receipt of events. So, for example, if a new account is created, Kill Bill triggers an `ACCOUNT_CREATION` event. The analytics plugin listens to this event and on receipt updates the analytics tables based on the Kill Bill tables. 
```

Comment:
Fixed, I've added a new line after the `This database is kept current in real-time.` instead of the `potentially changes only between major revisions of Kill Bill. ` as suggested. The `This database is kept current in real-time.` is related to the previous text, so I've kept it together.

Reply:
Ahh okay 👍 In this case, since it is just one sentence, let's leave it as it is, I will follow this for longer texts.